components/main.css

```diff
@@ -0,0 +1,259 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500&display=swap');
+:root{
+    font-family: 'Poppins', sans-serif;;
+}
+.logo a{
```

Comment:
you can use beautify file for formatting.

Reply:
I suggest prettier because beautify sometimes not arrange code properly.